pyxem/signals/correlation2d.py

```diff
@@ -72,10 +72,10 @@ def get_angular_power(self, inplace=False, **kwargs):
         power = self.map(corr_to_power, inplace=inplace, **kwargs)
         if inplace:
             self.set_signal_type("power")
-            fourier_axis = self.axes_manager.signal_axes[1]
+            fourier_axis = self.axes_manager.signal_axes[0]
```

Comment:
What's the context for these changes?

Reply:
These are just bug fixes. (And probably deserve their own PR as well)